DatabaseTemplate/Modules/get-populated-from-web.ps1

```diff
@@ -54,7 +54,7 @@ if ($fileName.EndsWith('.zip') -or $fileName.EndsWith('.7z')) {
     else {
         EnsureCommandIsAvailable "7z"
         $arguments = @("x", "$global:templateFolder/$fileName", "-o$global:templateDatabaseFolder")
-        7z @arguments
+        7z @arguments | Out-Null
```

Comment:
This script doesn't belong to the current group, but while testing this group I found that 7z output was being returned to the caller, so I forwarded its output to null to be consistent with the behavior in Windows.